JSDemos/Demos/DataGrid/CascadingLookups/Angular/app/app.component.ts

```diff
@@ -43,9 +44,9 @@ export class AppComponent {
     }
   }
 
-  setStateValue(rowData: any, value: any): void {
-    rowData.CityID = null;
-    (<any> this).defaultSetCellValue(rowData, value);
+  setStateValue(this: Column, newData: any, value: any, currentRowData: any) {
+    newData.CityID = null;
+    this.defaultSetCellValue(newData, value, currentRowData);
```

Comment:
this.defaultSetCellValue(newData, value, currentRowData) - according to [docs](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#setCellValue) only 2 params are needed, is it correct?

Reply:
Ah I see now it's more strict and we need to correct docs (?) 

Can use `Employee` for newData and currentRowData, e.g.

```
  setStateValue(this:Column, rowData: Employee, value: any, currentRowData:Employee): void {
    rowData.CityID = null;
    this.defaultSetCellValue(rowData, value, currentRowData);
  }
```

Reply:
As far as I see, our docs has already been updated. Regarding the second part, yes, thanks you